catalog/templates/base_generic.html

```diff
@@ -1,47 +1,88 @@
 <!DOCTYPE html>
 {% load i18n %}
 <html lang="en">
-<head>
-  {% block title %}<title>{% trans 'Local Library' %}</title>{% endblock %}
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
-  <!-- Add additional CSS in static file -->
-  {% load static %}
-  <link rel="stylesheet" href="{% static 'css/styles.css' %}">
-</head>
-<body>
-  <div class="container-fluid">
-    <div class="row">
-      <div class="col-sm-2">
-      {% block sidebar %}
-        <ul class="sidebar-nav">
-          <li><a href="{% url 'index' %}">{% trans 'Home' %}</a></li>
-          <li><a href="{% url 'books' %}">{% trans 'AllBooks' %}</a></li>
-          <li><a href="{% url 'authors' %}">{% trans 'AllAuthors' %}</a></li>
-        </ul>
-     {% endblock %}
-      </div>
-      <div class="col-sm-10 ">{% block content %}{% endblock %}
-      {% block pagination %}
-      {% if is_paginated %}
-        <div class="pagination">
-            <span class="page-links">
+    <head>
+        {% block title %}
+        <title>{% trans 'Local Library' %}</title>
+        {% endblock %}
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link
+            rel="stylesheet"
+            href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+            integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2"
+            crossorigin="anonymous"
+        />
+        <!-- Add additional CSS in static file -->
+        {% load static %}
+        <link rel="stylesheet" href="{% static 'css/styles.css' %}" />
+    </head>
+    <body>
+        <div class="container-fluid">
+        <div class="row">
+            <div class="col-sm-2">
+            {% block sidebar %}
+            <ul class="sidebar-nav">
```

Comment:
check lại indent file này nha
vd:
```suggestion
              {% if user.is_authenticated %}
                <li>{% trans "User:" %} {{ user.get_username }}</li>
                <li><a href="{% url 'my-borrowed' %}">{% trans "My Borrowed" %}</a></li>
                <li><a href="{% url 'logout'%}?next={{request.path}}">{% trans "Logout" %}</a></li>
              {% else %}
                ...
```